### ISSUE TITLE
Where did all the changes go?

### ISSUE THREAD

**Issue by cvrle77** [SubtitleEdit/subtitleedit#8390]
So, since the version 265:

Force stereo - completely gone, back to mono. 
It was properly working in v 264, both options with and without video (audio only) were giving normal stereo. 
The only issue was that it didn't save 'force stereo' box status.  There was absolutely no need to remove it.
It was 265 that had issues with stereo, and 263, and it seems I didn't unpack correct version, or forgot to test 264 to tell you it's ok, but I have it downloaded (luckily) OR, maybe you had 264 installed, and responded that everything was correct while I was getting bad results with 265.

![image](https://github.com/SubtitleEdit/subtitleedit/assets/16167396/77246646-a256-4e31-85b2-67dbe4d255fc)

Bug **fix** for superfast speech is also gone in new version, from this thread.
https://github.com/SubtitleEdit/subtitleedit/issues/8361#issuecomment-2108091232

https://drive.google.com/file/d/1XVlaxWY3mPi7evuF1TiUQuNkA2o96x65/view?usp=drive_link

Returning to 264, for TTS it was the best so far.

**Comment by BlueBlue7891** [comment]
Hello, can you send a copy of the beta264 file? I can no longer download this version. Thank you

**Comment by cvrle77** [comment]
[SubtitleEditBeta (13).zip](https://github.com/SubtitleEdit/subtitleedit/files/15371651/SubtitleEditBeta.13.zip)

**Comment by BlueBlue7891** [comment]
> [SubtitleEditBeta (13).zip](https://github.com/SubtitleEdit/subtitleedit/files/15371651/SubtitleEditBeta.13.zip)

Thank you very much.

**Comment by BlueBlue7891** [comment]
[SubtitleEditBeta (13).zip](https://github.com/SubtitleEdit/subtitleedit/files/15371651/SubtitleEditBeta.13.zip)
> 
Thank you very much.

I tested version 264, and it can indeed output stereo wav files, but after performing speech recognition, the content in the speed adjustment column in the audio preview window does not display correctly, showing all '-'. This affects my usage, so I updated to the latest SE Beta. Although it outputs mono, I can change the mono to stereo in video editing software (such as DaVinci)
